website/addons/dataverse/static/dataverseFangornConfig.js

```diff
@@ -254,25 +255,36 @@ function _fangornDataverseTitle(item, col) {
 
 function _fangornColumns(item) {
     var tb = this;
-    var selectClass = '';
     var columns = [];
     columns.push({
         data : 'name',
         folderIcons : true,
         filter : true,
-        css: selectClass,
         custom: _fangornDataverseTitle
     });
-
     if (tb.options.placement === 'project-files') {
         columns.push(
-            {
-                data: 'downloads',
-                filter: false,
-                css: ''
-            }
-        );
+        {
+            data  : 'size',
+            sortInclude : false,
+            filter : false,
+            custom : function() {return m('');}
+        });
     }
+    if (tb.options.placement === 'project-files') {
```

Comment:
Ditto [this comment](https://github.com/CenterForOpenScience/osf.io/pull/5817/files#r67015172)

>Minor: this looks like the same conditional as above; could you combine the 'downloads' and 'size' data to the same .push call?

Perhaps also consider DRYing this code up, but that's not a blocker.